.travis.yml

```diff
@@ -42,6 +42,9 @@ before_install:
     esac
 
     pip install pylint
+    pip install futures==3.1.1
+    pip install grpcio==1.4.0
```

Comment:
I'm skeptical as to whether this is the right way to declare dependencies. We already depend on several Python libraries from pypi—`bleach` and `markdown` among them—and we don't explicitly install any of them in the Travis config. (Installing `pylint` within Travis, on the other hand, makes sense, as it's only used as a separate test target.) I suspect that we instead want to add whichever packages end up being required to [third_party/python.bzl](https://github.com/tensorflow/tensorboard/blob/6fd136d8feaf11a81d1b287cc5029aad5e98ed85/third_party/python.bzl), and let Bazel do the dependency resolution automatically. (We don't do this for tensorflow and numpy, but they are exceptions, not the rule.) @jart please correct me if I'm wrong.